doc/classes/Object.xml

```diff
@@ -170,7 +200,7 @@
 			<param index="1" name="callable" type="Callable" />
 			<param index="2" name="flags" type="int" default="0" />
 			<description>
-				Connects a [param signal] to a [param callable]. Use [param flags] to set deferred or one-shot connections. See [enum ConnectFlags] constants.
+				Connects a [param signal] to a [param callable]. Optional [param flags] can be also added to configure the connection's behavior (see [enum ConnectFlags] constants).
 				A signal can only be connected once to a [Callable]. It will print an error if already connected, unless the signal was connected with [constant CONNECT_REFERENCE_COUNTED]. To avoid this, first, use [method is_connected] to check for existing connections.
 				If the callable's target is destroyed in the game's lifecycle, the connection will be lost.
 				[b]Examples with recommended syntax:[/b]
```

Comment:
Some of it shouldn't be outright moved elsewhere because other languages could need the example as a point of reference.

Honestly, the entire block is absurd. There's little to no point in showing the many different ways you can connect a signal, while only one is recommend. It's also easily the biggest description in the Class Reference. For its importance, that may be very understandable, but it's so large it may need its own PR.